### DISCUSSION TITLE
CM%40Cornell%20Tech/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#205] [Announcements]
# CM%40Cornell%20Tech/

None

https://opencs.app/CM%40Cornell%20Tech/

**Comment by Geeksongs** [comment]
这个项目消失了嘛？现在cornell tech没有这个项目了，而且还是两年，能不能1.5年毕业啊？

**Reply by xichenpan** [reply]
https://tech.cornell.edu/programs/masters-programs/jacobs-technion-cornell-dual-ms-connective-media/
